### PR TITLE
Move class dcs to proficiencies tab and shorten skills

### PR DESCRIPTION
**Pull request by CarlosFdez**
image

### CODE REVIEW

File: static/templates/actors/character/tabs/proficiencies.hbs [hunk 1]
```diff
@@ -156,4 +156,26 @@
             {{/with}}
         </ul>
     {{/if}}
+
+    <!-- Class DCs -->
+    {{#if classDCs.dcs}}
+        <header>{{#if (gt classDCs.dcs.length 1)}}{{localize "PF2E.Actor.Character.ClassDC.Plural"}}{{else}}{{localize "PF2E.Actor.Character.ClassDC.Label"}}{{/if}}</header>
```
Comment (CarlosFdez) [inline static/templates/actors/character/tabs/proficiencies.hbs:162]: I'm considering making it plural all the time. Let me tweak that right now.

**Comment by CarlosFdez** [comment]
Implemented using roll icons now

File: src/styles/actor/character/_proficiencies.scss [hunk 2]
```diff
@@ -21,59 +21,66 @@
...
+            font-size: var(--font-size-16);
+            line-height: 1em;
+            min-height: 2.25rem;
+            padding: 0 0.5rem 0 0.375rem;
 
-            a.skill-name,
-            a.modifier {
+            a:hover {
+                color: var(--primary);
+                cursor: pointer;
```
Comment (stwlam) [inline src/styles/actor/character/_proficiencies.scss:40]:
```suggestion
```

Comment (CarlosFdez) [reply src/styles/actor/character/_proficiencies.scss]: whoops, that slipped in. Fixed.

**Pull request merged**